samples/python/mser.py

```diff
@@ -1,56 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Move to snippets for MSER.

Reply:
@asmorkalov corresponding cpp sample [detect_mser.cpp](https://github.com/opencv/opencv/blob/5.x/samples/cpp/detect_mser.cpp ) is not in snippets. Should we still move python or keep it same as cpp

Reply:
yes, let's move both to snippets.